js/src/views/Connection/connection.js

```diff
@@ -17,7 +17,6 @@
 import React, { Component, PropTypes } from 'react';
 import { FormattedMessage } from 'react-intl';
 import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
 import ActionCompareArrows from 'material-ui/svg-icons/action/compare-arrows';
 import ActionDashboard from 'material-ui/svg-icons/action/dashboard';
 import HardwareDesktopMac from 'material-ui/svg-icons/hardware/desktop-mac';
```

Comment:
Actually, probably, yes. However I left those in there on purpose since the icons they are actually non-button icons where the rest are all meant for buttons.